FlagsmithClient/src/main/java/com/flagsmith/internal/FlagsmithAnalytics.kt

```diff
@@ -8,7 +8,7 @@ import android.util.Log
 import org.json.JSONException
 import org.json.JSONObject
 
-class FlagsmithAnalytics constructor(
+internal class FlagsmithAnalytics constructor(
```

Comment:
I do agree with this change, but I don't think it's necessary in this PR (?) and would _technically_ constitute a breaking change I think? On the off chance anyone has chosen to interact with the `FlagsmithAnalytics` class directly.

Reply:
It's up to you @matthewelwell just seemed sensible at the time when I reviewed the code

Reply:
Done